code/modules/food_and_drinks/kitchen_machinery/monkeyrecycler.dm

```diff
@@ -9,20 +11,31 @@
 	idle_power_usage = 5
 	active_power_usage = 50
 	circuit = /obj/item/circuitboard/machine/monkey_recycler
-	var/grinded = 0
-	var/required_grind = 5
-	var/cube_production = 1
+	var/stored_matter = 0
+	var/cube_production = 0.2
+	var/obj/machinery/computer/camera_advanced/xenobio/connected = list()
```

Comment:
?

Reply:
The list holds references to all the consoles that is connected to the recycler. It is used when the recycler is destroyed for some reason, removing the reference from the consoles.

Reply:
then it should be `var/list/connected`

Reply:
Fair enough, I will change it.